packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -338,7 +338,10 @@ contract L1StandardBridge_DepositERC20_Test is Bridge_Initializer {
             innerMessage
         );
 
-        // ERC20 Deposit Initiated event emitted by the StandardBridge Contract
+        // Should emit both the bedrock and legacy events
+        vm.expectEmit(true, true, true, true);
```

Comment:
It looks like there is an overload of `expectEmit` that also accepts the `address` as the final arg? Do you mind updating these to ensure the correct contracts are emitting the events?

Reply:
OK, done.